torch/nn/parallel/distributed.py

```diff
@@ -388,6 +393,14 @@ def __init__(
             "doesn't have any parameter that requires a gradient."
         )
 
+        assert (
+            device_ids is None or len(device_ids) <= 1
+        ), "device_ids can only be 1) None or an empty list,"
```

Comment:
Isn't it simpler to just say device_ids should be None or just contain a single element?

Reply:
Previously there is also a valid input which is an empty list. The error message is simplified now.